virttest/qemu_monitor.py

```diff
@@ -1239,6 +1239,25 @@ def get_migrate_capability(self, capability):
         value = re.search(pattern, capability_info, re.M).group(1)
         return value == "on"
 
+    def set_migrate_cache_size(self, value):
+        """
+        Set the cache size of migrate to value.
+
+        :param value: the cache size to set.
+        """
+        cmd = "migrate_set_cache_size"
+        self.verify_supported_cmd(cmd)
+        cmd += " %s" % value
+        return self.cmd(cmd)
+
+    def get_migrate_cache_size(self):
+        """
+        Get the xbzrel cache size. e.g. xbzrel cache size: 1048576 kbytes
+        """
+        cache_size_info = self.query("migrate_cache_size")
+        value = cache_size_info.split(":")[1].split()[0].strip()
```

Comment:
thanks coli, it doesn't need convert type, str is ok. easy to compare with the set value by funtion "set_migrate_cache_size"